book/website/collaboration/github/github-resources.md

```diff
@@ -1,12 +1,22 @@
-# Resources
+(cl-github-resources)=
+# Checklist and Resources
+
+## Checklist
 
 GitHub encourages collaboration practice in their community guidelines.
 The insights tab of your GitHub project provides a section called "Community" that includes a list of recommended documents that your project should have.
 Those include a README, Code of Conduct, Contributing guidelines, licence, and templates for issues and pull requests.
 
-![Community profile on GitHub](../../figures/community_profile.png)
+-```{figure} ../../figures/community_profile.png
+---
+height: 500px
```

Comment:
```suggestion
```

Reply:
Leaving out the height parameter makes sure that the image maintains its aspect ratio on smaller devices.

Reply:
@paulowoicho The examples in the [style guide](https://the-turing-way.netlify.app/community-handbook/style/style-figures.html) had the height parameter. Should we remove the height parameter from those examples?

Reply:
I think that would be a useful PR. We could add a note after the example that explains why the height parameter might not always be necessary. There's some explanation about that in the[ Maintaining Consistency chapter,](https://the-turing-way.netlify.app/community-handbook/consistency/consistency-formatting.html#check-4-use-myst-for-image-formatting) but it would be helpful to have that information in the style guide as well.